services/skus/credentials.go

```diff
@@ -186,6 +186,7 @@ type OrderWorker interface {
 // SignOrderCreds signs the blinded credentials
 func (service *Service) SignOrderCreds(ctx context.Context, orderID uuid.UUID, issuer Issuer, blindedCreds []string) (*OrderCreds, error) {
 
+	fmt.Println("!!! heyo here")
```

Comment:
lol, Ive not heard heyo before

Reply:
Fixed here: https://github.com/brave-intl/bat-go/pull/1714/commits/3c3c9aae70c02aef9f823d4bb2864104fd274108